samples/net/rpl-node/Makefile

```diff
@@ -1,15 +0,0 @@
-# Makefile - RPL node test application
```

Comment:
There is no CMakeLists.txt file for rpl-node sample.

Reply:
Are you familiar with this sample, or did you just happen to notice it?

Any assistance here would be appreciated.
https://github.com/zephyrproject-rtos/zephyr/issues/4791

Reply:
Ported.

Now it is broken in both KBuild and CMake, whereas before it was absent in CMake and broken in KBuild.

https://github.com/zephyrproject-rtos/zephyr/pull/4792